R/util.R

```diff
@@ -1711,3 +1780,82 @@ get_mode <- function(x, na.rm = FALSE) {
   ux <- unique(x)
   return(ux[which.max(tabulate(match(x, ux)))])
 }
+# Wrapper function that takes care of dplyr::group_by and dplyr::summarize as a single step.
+# @export
+summarize_group <- function(.data, grp_cols = NULL, grp_aggregations = NULL, ...){
+  library(dplyr)
+  if(length(grp_cols) == 0) {
+    .data %>% summarize(...)
+  } else {
+    groupby_args <- list() # default empty list
+    name_list <- list()
+    name_index = 1
+    # If group by columns and associated aggregation funcionts are provided,
```

Comment:
Typo: funcionts
Those functions are rather for categorization, and not for aggregation.
"group by columns" => "group_by columns"

Reply:
Fixed typo and updated comment.